v2/cmd/controller/app/flags.go

```diff
@@ -54,6 +56,8 @@ func ParseFlags(args []string) (Flags, error) {
 
 	// default here for 'MetricsAddr' is set to "0", which sets metrics to be disabled if 'metrics-addr' flag is omitted.
 	flagSet.StringVar(&metricsAddr, "metrics-addr", "0", "The address the metric endpoint binds to.")
+	flagSet.BoolVar(&secureMetrics, "secure-metrics", true, "Enable secure metrics. This will enable serving pprof endpoints and metrics securely using https")
```

Comment:
This probably should still be `secure-metrics` (the container cmdline arg). Just the Helm chart variable changes name?